commands-yml/commands/device/simulator/toggle-touch-id-enrollment.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: Toggle Touch ID Enrollment
+short_description: Toggle the simulator being [enrolled](https://support.apple.com/en-ca/ht201371) to accept touchId  (iOS Simulator only)
+
+description:
+  |
+    To enable this feature, the `allowTouchIdEnroll` desired capability must be set to true. When `allowTouchIdEnroll` is set to true
+    the Simulator will be enrolled by default, and the 'Toggle Touch ID Enrollment' changes the enrollment state.
+
+    This call will only work if Appium process or its parent application (e.g. Terminal.app or Appium.app) has access to Mac OS accessibility in System Preferences > Security & Privacy > Privacy > Accessibility list
```

Comment:
[Pedantic alert] `e.g.` should be followed by a comma (i.e., `(e.g., Terminal.app or Appium.app)`)